Sources/App/Core/Constants.swift

```diff
@@ -9,6 +9,8 @@ import Vapor
 
 
 enum Constants {
+    static let defaultGitlabPipelineLimit = 200
```

Comment:
I've set the fallback limit (if we don't have one set via secrets) a bit more conservative. Plan is still to go with 300 via the env variables.